pkg/consts/consts.go

```diff
@@ -15,4 +15,6 @@ const (
 
 	MaxBatchSize    = 100
 	MaxBatchTimeout = 60 * time.Second
+	// MaxEvents is the maximum number of events we can parse in a single batch.
+	MaxEvents = 5_000
```

Comment:
there's already a value defined here.
https://github.com/inngest/inngest/blob/d67b9e3a0cc170484066ba27c656ca1f38f2f50a/pkg/eventstream/eventstream.go#L15-L16

we should consolidate so we don't have 2 values.